src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -20,6 +20,11 @@
     #define MaxPower HighPower
 #endif
 
+#if defined(Regulatory_Domain_EU_CE_2400)
+    #undef MaxPower
+    #define MaxPower PWR_10mW
+#endif
+
```

Comment:
I think you will need to add `MinPower` and undef `HighPower` too.
Also, problematic is that Namimno Flash and HM ES24Pro, MinPower is defined as 25mW so they have no value for 10mW

Reply:
Can the Namimno Flash's amp even go down to 10mW? Or do manufacturers just have to label EU compliance (or not)